.travis.yml

```diff
@@ -39,7 +46,13 @@ matrix:
 before_script:
     - sudo ./scripts/configure_postgres.sh
 
-script: $SCRIPT
+script: 
+    - if [[ "${TRAVIS_CPU_ARCH}" == "arm64" ]]; then
+        sudo chmod +x $SCRIPT;
```

Comment:
I don't think you need this: you can `chmod +x` the script and commit it that way. Or is arm different in this respect?

Reply:
Updated as suggested.